### DISCUSSION TITLE
Github Discussions: Comment Started date/time should be Comment Updated date/time

### DISCUSSION THREAD

**Discussion by joebudi** [community/community#2996] [Discussions] [answered]
It would be great to have the comments metadata expose the Comment's updated date/time instead of Started time/date.

**Comment by evi-liu** [comment, marked as answer]
Hi @joebudi! You can currently view the comment updated time by clicking on the `edited by` dropdown on the upper right hand side of any edited comment.

![image](https://user-images.githubusercontent.com/8809849/102403798-a70c1600-3f9b-11eb-9263-9d25e1f21e54.png)
